### PR TITLE
Release/1.7.0

### PR DESCRIPTION
**Pull request by Trjegul84**

### CODE REVIEW

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/components/charts/LargeBarStackGraph.jsx [hunk 1]
```diff
@@ -44,6 +44,41 @@ const LargeBarStackGraph = (props) => {
 
   const keys = data.map((item) => String(item.key));
 
+    /**
+   * Get tooltip for graph component
+   *
+   * @param {String} id id for each bar
+   * @param {Object} allData transformed data with all information needed
+   * @param {String} color color for each category inside a bar
+   * @returns {func} tooltip for component
```
[the remark this answers is not in the document]
Comment (Trjegul84) [reply src/components/charts/LargeBarStackGraph.jsx:53]: I've also found getToolTip returning func in SmallBarStackGraph. Should I change both to `@returns {object}` ?

[the remark this answers is not in the document]
Comment (erikasv) [reply src/components/charts/LargeBarStackGraph.jsx:53]: yes please

[the remark this answers is not in the document]
Comment (Trjegul84) [reply src/components/charts/LargeBarStackGraph.jsx]: Done in the  feature/fixIndentation PR

File: src/components/charts/LargeBarStackGraph.jsx [hunk 2]
```diff
@@ -44,6 +44,41 @@ const LargeBarStackGraph = (props) => {
 
   const keys = data.map((item) => String(item.key));
 
+    /**
```
[the remark this answers is not in the document]
Comment (Trjegul84) [reply src/components/charts/LargeBarStackGraph.jsx]: Fixed in the PR of feature/fixIndentation branch

**Review by erikasv — APPROVED** [review]

**Review by alegnaaived — APPROVED** [review]

**Review by cazapatamar — APPROVED** [review]

**Pull request merged**